eng/versioning/version_client.txt

```diff
@@ -454,4 +454,4 @@ unreleased_com.azure:azure-core-test;1.22.0-beta.1
 # Format;
 # beta_<groupId>:<artifactId>;dependency-version
 # note: Released beta versions will not be manipulated with the automatic PR creation code.
-
+beta_com.azure:azure-data-appconfiguration;1.5.0-beta.3
```

Comment:
1.5.0-beta.3 hasn't released.
 You can either use in `unreleased` section
unreleased_com.azure:data-appconfiguration;1.5.0-beta.3 

or change the version to 1.5.0-beta.2